channels-dvr.json

```diff
@@ -9,9 +9,10 @@
     "pkgs": [
         "ftp/curl",
         "security/ca_root_nss",
-        "security/sudo"
+        "security/sudo",
+        "www/chromium"
     ],
-    "packagesite": "http://pkg.cdn.trueos.org/iocage/unstable",
+    "packagesite": "http://pkg.FreeBSD.org/${ABI}/latest",
```

Comment:
If you change the "packagesite", you also need to change the value of "fingerprint" to b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438

Reply:
Thank you! I was wondering what that checksum was for.